manta-trusted-setup/src/groth16/ppot/serialization.rs

```diff
@@ -253,10 +253,10 @@ impl Serializer<G1Affine, G1> for PpotSerializer {
             // Final result will be reversed, so this is like modifying first byte
             res[31] |= 1 << 6;
         } else {
-            let mut temp_writer = &mut res[..];
+            let temp_writer = &mut res[..];
```

Comment:
Why can we do this?

Reply:
I think it originally was an unnecessary `mut` because `temp_writer` is a mutable reference, but remains a mutable reference to the thing. So `temp_writer` itself is never mutated, just the data it references. I guess this is something clippy recently started noticing.